oct/ansible/oct/callback_plugins/log_results.py

```diff
@@ -1,16 +1,44 @@
 # coding=utf-8
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from codecs import open
-from datetime import datetime
-from os import environ, makedirs
-from os.path import exists, join, sep
-from shutil import rmtree
-from tempfile import mkdtemp
-from traceback import format_exc
+import contextlib
+from functools import wraps
+import logging
+import os
+from os.path import join
 
 from ansible.plugins.callback import CallbackBase
-from yaml import dump
+import structlog
+from structlog.stdlib import LoggerFactory
+
+
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,
+        structlog.processors.TimeStamper(fmt="iso"),
+        structlog.processors.format_exc_info,
+        structlog.processors.UnicodeDecoder(),
+        structlog.processors.JSONRenderer(),
+    ],
+    context_class=dict,
+    logger_factory=structlog.stdlib.LoggerFactory(),
+    wrapper_class=structlog.stdlib.BoundLogger,
+    cache_logger_on_first_use=True)
+handler = logging.FileHandler(
+    join(os.environ.get('ANSIBLE_LOG_ROOT_PATH'), 'log.txt'), 'a')
```

Comment:
With the code as it is on this PR, the log file would be at

    ~/.config/origin-ci-tool/logs/log.txt

Can we change it to a single log file, e.g.

    ~/.config/origin-ci-tool/log.txt

or would it impact other places that look for the logs in this location?

There doesn't seem to be a reference to this path on this repository, but I
don't know about other systems.